openmc/tallies.py

```diff
@@ -13,11 +13,12 @@
 
 import numpy as np
 
-from openmc import Mesh, Filter, Trigger, Nuclide
-from openmc.arithmetic import *
+from openmc import Filter, Trigger, Nuclide
+from openmc.arithmetic import CrossScore, CrossNuclide, CrossFilter, \
```

Comment:
I hear you. It's a little more verbose but with the benefit that you know where exactly those classes come from without having to `grep` your source directory or rely on a fancy IDE. The other acceptable (as far as pylint is concerned) alternative is to `import openmc.arithmetic` and then explicitly say `openmc.arithmetic.CrossScore`, but that gets a little annoying with long names.